### DISCUSSION TITLE
Half-Life Running on a MacBook Air M1 on Linux

### DISCUSSION THREAD

**Discussion by m1m1k4tz** [ptitSeb/box64#2107] [Show and tell]
https://www.youtube.com/watch?v=4EscY1K7Eeo

**Comment by ptitSeb** [comment]
You should really try box32 with regular wine. some stuffs work, and when they do, that are fast.

**Reply by m1m1k4tz** [reply]
I tried with just the 32 bit version and it wouldn’t detect the drivers I’m guessing I would have to go back and compile a shared wow version and that would probably work

**Reply by ptitSeb** [reply]
what driver? regular wine  or 64bits wine  (the one I use, not wow64 one)  should just work, but no vulkan will be available on 32bits, only OpenGL.

**Reply by m1m1k4tz** [reply]
I couldn’t even get winecfg to work with kroneks 32 bit version sadly

**Reply by ptitSeb** [reply]
Use the 64bits version. Not the Wow64 one, the regular 64bits version. So wineserver and other stuffs will use box64, will wine and 32bits parts will use box32.

**Reply by m1m1k4tz** [reply]
Yeah I think that’s the best way to do it. I need to go back and grab an old version of the script that compiles wine for a shared wow64 build not the new version. Ideally, Steam would do it all for me under box32.

**Reply by ptitSeb** [reply]
I used this one https://github.com/Kron4ek/Wine-Builds/releases/download/9.20/wine-9.20-staging-tkg-amd64.tar.xz worked fine for me.

**Reply by m1m1k4tz** [reply]
Okay, I’ll try that out instead

**Reply by m1m1k4tz** [reply]
I actually get the same no driver issue for some reason with that build. I submitted a ticket with the log.

**Reply by m1m1k4tz** [reply]
Okay, I just tried Fallout 3 which has no DRM and I cant get it to work because regedit and reg wont save the registry keys I add even with a reg file

**Reply by ptitSeb** [reply]
I don't reproduce this issue on my side with latest box64/box32 (using wine 9.20-tkg, or 9.22-tkg)
